srcalc/src/main/java/gov/va/med/srcalc/service/ModelInspectionService.java

```diff
@@ -24,4 +24,18 @@
      * @return an ImmutableCollection, in arbitrary order
      */
     public ImmutableCollection<RiskModel> getAllRiskModels();
+
+    
+    /**
+     * Returns the {@link RiskModel} with the given ID.
+     * @return RiskModelS
+     */
+    public RiskModel getRiskModelForId(int modelId);    
+    
+    /**
+     * Saves the name of the {@link RiskModel} 
```

Comment:
This doc comment is inaccurate: this will actually save the whole RiskModel object, including all of its terms.